ChangeLog.md

```diff
@@ -17,6 +17,7 @@ See docs/process.md for how version tagging works.
 
 Current Trunk
 -------------
+- Default DISABLE_DEPRECATED_FIND_EVENT_TARGET_BEHAVIOR to 1.
```

Comment:
Please add some details here to help people, like the PR number here, but even that isn't enough. Can we briefly summarize what the likely errors people will hit look like, and what the resolution to this is? I assume that's documented somewhere?

Reply:
OK. I had assumed that the information in settings.js would have been enough, but I'll put that here.